### DISCUSSION TITLE
翻译 OS 标签

### DISCUSSION THREAD

**Discussion by linonetwo** [BramChen/tw5-docs#6] [Ideas]
<img width="1600" alt="8593ED67FE155D1976736F3A3BA873B6" src="https://user-images.githubusercontent.com/3746270/169205699-a043643a-9a3f-4163-8684-0570b0ae4087.png">

这里的 Standalone App 如果也翻译为汉语会更好？

还有我也不确定 Standalone App 是归到 OS 还是 Browser 好，我就先放 Browser 了，你觉得是作为 OS 更好吗？

**Comment by linonetwo** [comment]
最近在论坛讨论，其实建议新用户来装 app ，这样可以避免用户流失

**Comment by BramChen** [comment]
> 还有我也不确定 Standalone App 是归到 OS 还是 Browser 好，我就先放 Browser 了，你觉得是作为 OS 更好吗？

就是手誤，感謝指正。(我覺得最好是獨立一個分類)

**Reply by linonetwo** [reply]
这个只是过渡方案，之后要改成 wizard （步骤助手？）的 https://github.com/Jermolene/TiddlyWiki5/pull/4730#issuecomment-650551710
